Sources/SwiftCrossUI/Environment/EnvironmentValues.swift

```diff
@@ -183,6 +183,15 @@ public struct EnvironmentValues {
         )
     }
 
+    /// Closes the current window.
```

Comment:
Reword to `enclosing window`, as I feel that `current` could be interpreted to mean 'primary' or 'focused'.